kube-burner-workload/openshift/openflow-tracker/openflow-tracker.py

```diff
@@ -155,14 +157,17 @@ def main():
         "workload": "network-policy-perf",
         "uuid": uuid,
         "source_name": node_name,
-        "threshold": threshold,
+        "convergence_period": convergence_period,
+        "convergence_timeout": convergence_timeout,
```

Comment:
AFAIK, grafana dashboard does not use these fields, so no change is needed on that end, right?